components/SocialIcons/SocialSvgComponents.tsx

```diff
@@ -0,0 +1,66 @@
+import { SVGProps } from "react";
+
+export const FacebookIcon = (props: SVGProps<SVGSVGElement>) => (
+  <svg width="40" height="40" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg" {...props}>
+    <circle cx="20" cy="20" r="20" fill={props.fill} />
+    <path
+      fill="white"
```

Comment:
These SVGs should be made accessible. Some possible approaches: https://css-tricks.com/accessible-svgs/

Reply:
Thanks for the resource! I went with approach 2, but would like some feedback on the alt text I used in components/svgs/{icon}.tsx